support-frontend/assets/components/product/PaperProductCard.tsx

```diff
@@ -0,0 +1,137 @@
+import { css } from '@emotion/react';
```

Comment:
Can see a lot of similarities with `productOption.tsx`.  Am guessing, in the end, if successful, this should stay and `productOption.tsx` will go.